docs/inference_helpers/inference_sdk.md

```diff
@@ -343,6 +343,37 @@ CLIENT = InferenceHTTPClient(
 CLIENT.unload_all_models()
 ```
 
+## Inference `deployments`
+
+!!! tip
+
+    This feature is in `alpha` preview. We encourage to experiment and reach us out with issues spotted.
```

Comment:
Should be "We encourage you to experiment and reach out to us with issues spotted."

Reply:
👍